cpp/src/io/parquet/page_enc.cu

```diff
@@ -1735,6 +1809,11 @@ dremel_data get_dremel_data(column_view h_col,
 
   stream.synchronize();
 
+  size_t max_vals_size = 0;
+  for (size_t l = 0; l < column_offsets.size(); ++l) {
+    max_vals_size += column_ends[l] - column_offsets[l];
+  }
+
```

Comment:
`thrust::reduce()`?  Maybe overkill for such small numbers.

Reply:
I tried this but the iterator code was more confusing to me personally so I reverted it.